models/marts/ledger_current_state/accounts_current.sql

```diff
@@ -60,9 +60,31 @@ with
         {% endif %}
     )
 
+    , account_date as (
```

Comment:
nit: accounts is pretty small but I'm pretty sure we could create an int_* for account create time instead of running this CTE. It's probably also possible to only look up relevant account_ids in incremental runs instead of all account_ids

Reply:
Yeah, I agree. I created a ticket to research/refactor: https://stellarorg.atlassian.net/browse/HUBBLE-498